### PR TITLE
fix: prefix not using global settings

### PR DESCRIPTION
**Pull request by lee-chase**
Carbon prefix should be retrieved from the global settings not directly from Carbon.

#### Changelog

m packages/experimental/src/components/PageHeader/ButtonSetWithOverflow.js 
m packages/experimental/src/components/PageHeader/TempComboButton.js

### CODE REVIEW

File: packages/experimental/src/components/PageHeader/TempComboButton.js [hunk 1]
```diff
@@ -34,7 +32,7 @@ export const TempComboButton = ({ buttons, className }) => {
       menuOptionsClass={`${blockClass}--options`}
       renderIcon={() => (
         <div
-          className={`${blockClass}--trigger  ${prefix}--btn ${prefix}--btn--primary ${prefix}--btn--field`}>
+          className={`${blockClass}--trigger  ${carbonPrefix}--btn ${carbonPrefix}--btn--primary ${carbonPrefix}--btn--field`}>
```
Comment (davidmenendez) [inline packages/experimental/src/components/PageHeader/TempComboButton.js:35]: can you use the `cx` function for this please?

**Review by davidmenendez — CHANGES_REQUESTED** [review]
just one comment

**Review by SimonFinney — COMMENTED** [review]
LGTM pending resolution of https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/331/files#r574049066

Additionally, what's the position on using the 'ready to merge' label? If there are multiple reviews on a PR, this will still trigger if one has approved and the other is still pending.

**Comment by lee-chase** [comment]
> LGTM pending resolution of https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/331/files#r574049066
> 
> Additionally, what's the position on using the 'ready to merge' label? If there are multiple reviews on a PR, this will still trigger if one has approved and the other is still pending.

I've tended to add 'ready to merge' when I am happy, but it probably should not merge if there are outstanding change requests or multiple reviews required.

**Review by davidmenendez — APPROVED** [review]

**Pull request merged**